### PR TITLE
Release 5.5.1 Lacework Agent Charts

### PR DESCRIPTION
**Pull request by cirego**

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ entries:
   admission-controller:
   - apiVersion: v2
     appVersion: 0.0.1
-    created: "2022-05-05T22:09:37.397718-07:00"
```
Comment (nschmeller) [inline index.yaml:6]: Looks like we updated time stamps of charts that are not in this PR?

**Review by nschmeller — APPROVED** [review]

File: index.yaml (hunk 1 above)
Comment (cirego) [reply index.yaml:6]: Yeah, sadly, `helm` doesn't have a flag to avoid updating these timestamps.

File: lacework-agent/templates/daemonset.yaml [hunk 2]
```diff
@@ -96,6 +96,9 @@ spec:
             readOnly: true
           - name: podinfo
             mountPath: /etc/podinfo
+{{- if kindIs "string" .Values.laceworkConfig.serviceAccountName }}
+      serviceAccountName: {{ .Values.laceworkConfig.serviceAccountName | quote }}
+{{- end}}
```
Comment (joebowbeer) [inline lacework-agent/templates/daemonset.yaml:101]: @cirego Is this documented anywhere?

Why isn't there a corresponding template to create a service account with this name?

I notice that by default, Lacework uses the default service account, which is something that CIS Benchmark recommends against.

However, if I specify serviceAccountName, then I need to create that service account myself.

Comment (cirego) [reply lacework-agent/templates/daemonset.yaml:101]: Hi @joebowbeer, no, we have not documented this outside of Lacework as this functionality was only required for OpenShift. As for why we didn't add this functionality before? I commented in more detail on #105.

If you would like us to add a flag to auto-create the Service Account, even for agents that don't access the K8s API Server, please let us know!

**Pull request merged**